allauth/account/app_settings.py

```diff
@@ -190,6 +190,15 @@ def LOGIN_ON_EMAIL_CONFIRMATION(self):
         return self._setting('LOGIN_ON_EMAIL_CONFIRMATION', True)
 
     @property
+    def LOGIN_CALLBACK_PROXY(self):
+        return self._setting('LOGIN_CALLBACK_PROXY', '')
+
+    @property
+    def LOGIN_PROXY_REDIRECT_WHITELIST(self):
+        return filter(len,
```

Comment:
prefer to use list comprehensions with an "if" clause, it's a little clearer to those that don't have a functional background what's going on